demo/content/peeking.md

```diff
@@ -163,3 +163,34 @@ This also works well without looping. Note how the peeking card switches to the
   <slide :index='5'></slide>
 </ssr-carousel>
 ```
+
+## Supply your own overflow:hidden
+
+Add the `overflow-visible` prop to supply your own `overflow: hidden`, for cases where you want to prevent the width of the carousel from expanding but do want the carousel to extend to the edges of the page.  It makes the most sense with a big `peek-right` that hints at other slides.
```

Comment:
@brokenhd I added a demo of the approach we came up with today here: https://deploy-preview-87--vue-ssr-carousel.netlify.app/peeking, because I bet we'll be applying this on other projects as well.